src/features/customerView/boatsCard/__tests__/__snapshots__/BoatsCard.test.tsx.snap

```diff
@@ -1,3 +1,3 @@
on\\"><div class=\\"sectionHeadWithButton\\"><h4 class=\\"text standard h4 m title\\">VENEEN TIEDOT</h4><button type=\\"button\\" class=\\"title\\"><span class=\\"text brand span\\">Muokkaa</span></button></div><section class=\\"body\\"><div class=\\"labelValuePair\\"><span class=\\"label standard\\">Veneen tyyppi:</span><span class=\\"value left\\">Suuri alus (yli 20t)</span></div><div class=\\"labelValuePair\\"><span class=\\"label standard\\">Rekisterinumero:</span><span class=\\"value left\\">TRLR-666</span></div></section></article><article class=\\"section\\"><section class=\\"body\\"><div class=\\"labelValuePair\\"><span class=\\"label standard\\">Veneen leveys:</span><span class=\\"value left\\">4,0 m</span></div><div class=\\"labelValuePair\\"><span class=\\"label standard\\">Veneen pituus:</span><span class=\\"value left\\">15,0 m</span></div><div class=\\"labelValuePair\\"><span class=\\"label standard\\">Veneen syväys:</span><span class=\\"value left\\">2,0 m</span></div><div class=\\"labelValuePair\\"><span class=\\"label standard\\">Veneen paino:</span><span class=\\"value left\\">15 000 kg</span></div></section></article><article class=\\"section\\"><section class=\\"body\\"><div class=\\"labelValuePair\\"><span class=\\"label standard\\">Nimi:</span><span class=\\"value left\\">DevilsDriver</span></div><div class=\\"labelValuePair\\"><span class=\\"label standard\\">Merkki:</span><span class=\\"value left\\">Trawl 77</span></div></section></article><article class=\\"section\\"><section class=\\"body\\"><div class=\\"labelValuePair\\"><span class=\\"label standard\\">Käyttövoima:</span><span class=\\"value left\\">Bensin</span></div><div class=\\"labelValuePair\\"><span class=\\"label standard\\">Runkomateriaali:</span><span class=\\"value left\\">Aluminium</span></div><div class=\\"labelValuePair\\"><span class=\\"label standard\\">Käyttötarkoitus:</span><span class=\\"value left\\">For fun</span></div></section></article><article class=\\"section\\"><section class=\\"body\\"><div class=\\"boatCertificate\\"><div class=\\"boatCertificateLabel\\">Katsastustodistus:</div><div class=\\"boatCertificateValues\\"><div><span class=\\"text critical span\\">Tiedosto puuttuu</span></div><div><span class=\\"text standard span\\">Voimassa 04.09.2099 asti</span></div><div>Tarkastettu 28.04.2020</div><div>Jukka Virtanen</div></div></div></section></article><article class=\\"section\\"><section class=\\"body\\"><div class=\\"boatCertificate\\"><div class=\\"boatCertificateLabel\\">Vakuutuskirjaote:</div><div class=\\"boatCertificateValues\\"><div><a class=\\"link default\\" href=\\"https://venepaikka-api.test.hel.ninja/media/boats/2c736de2-5659-46ea-ad3f-9c1364307d7d/dummy_pdf2.pdf\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\">dummy_pdf2.pdf</a></div><div>Tarkastettu 28.04.2020</div></div></div></section></article></div><div class=\\"cardBody createBoat\\"><button><span class=\\"text brand span\\">Lisää uusi</span></button></div></div>"`;
```

Comment:
This snapshot could be switched to `.render()`

Reply:
Changed it.